### ISSUE TITLE
Currency Converter

### ISSUE THREAD

**Issue by viktorahlstrom** [karan/Projects#74]
I've already made something for this, and I think it would be interesting to see how people would do it in other languages. :smile:

**Comment by JayDex18** [comment]
Does this fetch live exchange rates or has a specific refresh interval ?

**Comment by viktorahlstrom** [comment]
Yes, it should fetch live exchange rates. An example of it in [Bash] shell script is [here](https://github.com/viktorahlstrom/all/blob/master/scripts/cconvert.sh). In my case, I fetched exchange rates from Google's Finance API using `wget`.

**Comment by sbelidhe** [comment]
Please refer to code [CurrencyConverter](https://github.com/sbelidhe/karangoelprojects/blob/master/CurrencyConverter.java) written in java using currencylayer API as both Yahoo and Google has decommissioned the API call which was used before for currency conversion. Also note that to parse JSON data used json-simple-1.1 library

**Comment by charleslarnold** [comment]
I wrote a currency coverter in python
